app/editor/src/features/content/form/ContentActions.tsx

```diff
@@ -124,6 +125,10 @@ export const ContentActions: React.FC<IContentActionsProps> = ({
               onChange={(e) => {
                 const checked = e.currentTarget.checked;
                 if (!checked) setFieldValue(field('value', index), '');
+                else {
+                  const weekDay = moment(values.publishedOn).weekday();
+                  setFieldValue(field('value', index), `${weekDay === 0 || weekDay === 6 ? 3 : 5}`);
```

Comment:
One issue with this logic is for those who have their local configuration set to have Monday as the first day of the week.  I don't think that will be relevant for editors however.